client/js/screens/Home/HomeContainer.js

```diff
@@ -1,27 +1,44 @@
 import React, {Component} from 'react';
 import {compose} from 'recompose';
-import {graphql, Query} from 'react-apollo';
+import {graphql} from 'react-apollo';
 import {
   QUERY_USER,
   QUERY_ALL_EVENTS,
   QUERY_EVENT_THIS_WEEK,
 } from '../../apollo/queries';
-import {StatusBar, Text, View, LayoutAnimation} from 'react-native';
-import {VibrancyView} from '@react-native-community/blur';
+import {withCollapsible} from 'react-navigation-collapsible';
+import {StatusBar, View} from 'react-native';
 import Home from './Home';
 import {getParamFromParent} from '../../lib/paramFromParent';
 import {calculateRatingScore} from '../../lib/calculateRatingScore';
 import PropTypes from 'prop-types';
 
 class HomeContainer extends Component {
-  static navigationOptions = props => {
+  static navigationOptions = ({navigation, navigationOptions}) => {
     return {
       headerTransparent: true,
+      headerStyle: {
+        backgroundColor: 'transparent',
+      },
+      headerBackground: () => {
+        return (
+          <View>
+            <StatusBar barStyle="light-content" />
+          </View>
```

Comment:
just return `StatusBar`. Please remove unnecessary wrapping of `View` component.

Reply:
Done